cf-key/cf-key.c

```diff
@@ -354,6 +354,7 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             }
 
         case 'C':
+            OPTIONAL_ARGUMENT_IS_PRESENT;
             if (!GenericAgentConfigParseColor(config, optarg))
```

Comment:
Why not `if (OPTIONAL_ARGUMENT_IS_PRESENT && (!GenericAgentConfigParseColor(config, optarg))`?